src/main/java/org/spongepowered/api/text/format/TextStyle.java

```diff
@@ -445,18 +445,18 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        if (this.bold.isPresent() || this.italic.isPresent() || this.underline.isPresent() || this.strikethrough.isPresent() || this.obfuscated
-                .isPresent()) {
+        if (isEmpty()) {
             return Objects.toStringHelper(this)
-                    .add("bold", this.bold)
-                    .add("italic", this.italic)
-                    .add("underline", this.underline)
-                    .add("strikethrough", this.strikethrough)
-                    .add("obfuscated", this.obfuscated)
+                    .addValue("NONE")
```

Comment:
Maybe it would make sense to use only `omitNullValues()` so that an empty `TextStyle` would be simply `TextStyle{}`